src/types.ts

```diff
@@ -525,32 +732,231 @@ export interface InitialAccountCredential {
     contents: InitialCredentialDeploymentValues;
 }
 
-export interface BakerReduceStakePendingChange {
-    change: 'ReduceStake';
-    newStake: bigint;
-    epoch: bigint;
+export enum StakePendingChangeTypeV0 {
+    ReduceStake = 'ReduceStake',
+    RemoveStake = 'RemoveBaker',
+}
+
+export enum StakePendingChangeTypeV1 {
+    ReduceStake = 'ReduceStake',
+    RemoveStake = 'RemoveStake',
```

Comment:
Would it be cleaner to have 1 enum like:
```suggestion
export enum StakePendingChangeType {
    ReduceStake = 'ReduceStake',
    RemoveStakeV0 = 'RemoveBaker',
    RemoveStakeV1 = 'RemoveStake',
```